internal/runner/runner.go

```diff
@@ -15,8 +15,15 @@ import (
 // OsExit is use here in order to simplify testing
 var OsExit = os.Exit
 
+type Style string
```

Comment:
```suggestion
type OutputStyle string
```

to make it more explicit